iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -962,15 +974,12 @@ public override async Task SendTwinPatchAsync(TwinCollection reportedProperties,
                 var websocket = new ClientWebSocket();
                 websocket.Options.AddSubProtocol(WebSocketConstants.SubProtocols.Mqtt);
 
-                // Check if we're configured to use a proxy server
-                IWebProxy webProxy = settings.Proxy;
-
                 try
                 {
-                    if (webProxy != DefaultWebProxySettings.Instance)
+                    if (_webProxy != DefaultWebProxySettings.Instance)
```

Comment:
could it be null or invalid either?

Reply:
I can add a null check to the helper function that will replace this line, sure